sirepo/base_pkconfig.py

```diff
@@ -15,7 +15,15 @@ def beta():
 
 
 def dev():
-    return {}
+    return {
```

Comment:
I like that you read up on pkconfig. Unfortunately, I am thinking of getting rid of base_config.py, because no one uses it, and we tend to use dynamic defaults ([example in pykern](https://github.com/radiasoft/pykern/blob/582f61bbc3eead69b01b26c17759f6c5f0efea13/pykern/pkcli/github.py#L244)). I would like your opinion on that.

In BOP, we used this committed config in a central file quite a bit, and it's fine, but there's just less config in Sirepo, and more hard coding -- for better or worse.

Reply:
The dynamic defaults version seems fine to me. It reduces the number of files you have to cross-reference to figure out what's going on with a given option, which is nice. (OTOH I guess it increases the number of files you have to look at to figure out what's going on with a given channel.)

I'm happy to do it either way; I was just trying to make a plausible guess about what would fit in with your style :-)

Reply:
Let's go with "dynamic defaults". While you are in base_config.py, put a comment somewhere that says don't use this file. That will help others. Thanks.